cmd/emailreport/main.go

```diff
@@ -75,6 +76,7 @@ func main() {
 	var metricsdir string
 
 	flag.StringVar(&confFile, "f", "", "Configuration file")
+	flag.StringVar(&comments, "c", "", "comments/suggestions")
```

Comment:
You should probably add this to the README.md as well :-)

Reply:
you are right :)